codespell_lib/data/dictionary.txt

```diff
@@ -2069,7 +2069,11 @@ authoritiers->authorities
 authorititive->authoritative
 authoritive->authoritative
 authos->authors, autos,
+authror->author
+authrored->authored
 authrorities->authorities
+authrorization->authorization
```

Comment:
Also "authorisation" in BE

Reply:
I'll make a new entry `authrorisation->authorisation`

Reply:
d1c4b2e